File: matlab_tests/test_switch.m
% Test switch-case statements
function operation = test_switch(choice)
    switch choice
        case 1
            operation = 'addition';
        case 2
            operation = 'subtraction';
        case 3
            operation = 'multiplication';
        case 4
            operation = 'division';
        otherwise
            operation = 'unknown';
    end
end

File: matlab_tests/test_switch.py
import numpy as np

# Test switch-case statements

def test_switch(choice):
    if choice == 1:
        operation = 'addition'
    elif choice == 2:
        operation = 'subtraction'
    elif choice == 3:
        operation = 'multiplication'
    elif choice == 4:
        operation = 'division'
    else:
        operation = 'unknown'
    
    return operation
